### ISSUE TITLE
Issue using library

### ISSUE THREAD

**Issue by mickeldebs** [kpn/nuxt-pino-log#3]
Hi, thanks for you work.
I'm currently new to nuxt and having issues setting this up.
Made the changes required and I'm getting this error:
![image](https://github.com/kpn/nuxt-pino-log/assets/76426689/7ca9450d-ea73-49d2-8579-6e0e82bbfc0a)
Don't know if I'm missing something

**Comment by woile** [comment]
Did you check the example? https://github.com/kpn/nuxt-pino-log/blob/main/example/nuxt.config.js
Which version of nuxt are you using?
Can you provide a sample repo?

Regards

**Comment by mickeldebs** [comment]
Sure, I'm using nuxt: 3.4.3
here is a sample repo that replicates the issue:
[Repo](https://github.com/mickeld23/sample-nuxt-app)

**Comment by woile** [comment]
I don't think this repo support nuxt `v3`. It was designed for `v2`, and at the time `v3` was quite green, feel free to send a PR with an update. Apologies in advance
